### DISCUSSION TITLE
Building FIPS-compliant NodeJS on Windows

### DISCUSSION THREAD

**Discussion by seidhkona** [nodejs/help#4929] [Q&A]
Hi everyone,
I'm new to the topic of building Node, sorry if this has been asked before and I missed it. I'm trying to research if it's possible to build node with FIPS-compliant OpenSSL on windows? I see the instructions on how to do it for Linux, but nothing for Windows. I appreciate any advice. Thanks!

**Comment by liuxingbaoyu** [comment]
`.\vcbuild.bat --config_flags "--shared-openssl --shared-openssl-libpath=F:\openssl --shared-openssl-includes=F:\openssl\include --shared-openssl-libname=crypto,ssl" debug`

I use this command to build nodejs with custom openssl.

You can add the fips parameter like this.
